packages/rest-typings/src/v1/teams/TeamsListRoomsAndDiscussions.ts

```diff
@@ -0,0 +1,24 @@
+import type { ITeam } from '@rocket.chat/core-typings';
+
+import type { PaginatedRequest } from '../../helpers/PaginatedRequest';
+import { ajv } from '../Ajv';
+
+export type TeamsListRoomsAndDiscussionsProps = PaginatedRequest<{
+	teamId: ITeam['_id'];
+	filter?: string;
+}>;
+
+const TeamsListRoomsAndDiscussionsPropsSchema = {
+	type: 'object',
+	properties: {
+		teamId: { type: 'string' },
```

Comment:
Could we also accept `teamName` here? wdyt? Seems like most `teams` endpoints do so